src/main/eo/org/eolang/math/series.eo

```diff
@@ -0,0 +1,67 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2021-2022 Max Trunnikov
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
++alias org.eolang.collections.list
++home https://github.com/objectionary/eo-math
++package org.eolang.math
++rt jvm org.eolang:eo-math:0.0.0
++version 0.0.0
+
+# Series
+[arr] > series
+  arr > @
+
+  # @todo #36:30 min. The array may consist of objects of any type. `max` and
+  #  `min` objects should work only with an array of numbers.
+  # Find max element in the array
+  [] > max
+    list ^ > self-as-list!
+
+    if. > @
+      self-as-list.is-empty
+      []
+        "Can't get max of empty array" > msg
```

Comment:
@maxonfjvipon I think we can use [error](https://news.eolang.org/2022-07-18-error-and-try-catch.html) object here

Reply:
Done!